app/components/dashboard/run/left-panel/component.js

```diff
@@ -165,5 +166,63 @@ export default Component.extend(FullScreenMixin, {
         denyDelete() {
             return true;
         },
+        
+        exportTale(id) {
+          let self = this;
+          // Asks the girder endpoint for a zipped Tale
+          // This code was adapted from
+          // https://stackoverflow.com/questions/16086162/handle-file-download-from-ajax-post
+          const token = self.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          var xhr = new XMLHttpRequest();
+          xhr.open('GET', url, true);
+          xhr.responseType = 'arraybuffer';
+          xhr.setRequestHeader("Girder-Token", token);
+          self.set('showDimmer', true);
+          xhr.onload = function () {
+              if (this.status === 200) {
+                self.set('showDimmer', false);
+                  var filename = "";
+                  var disposition = xhr.getResponseHeader('Content-Disposition');
+                  if (disposition && disposition.indexOf('attachment') !== -1) {
+                      var filenameRegex = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/;
+                      var matches = filenameRegex.exec(disposition);
+                      if (matches != null && matches[1]) filename = matches[1].replace(/['"]/g, '');
```

Comment:
I don't think we need this filename handling..

Since we control the name of the file that will be downloaded, which is currently always a Tale ID, we won't ever have quotes to strip out